ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -431,3 +431,27 @@ def get_deviceset_count():
     return int(sc.get().get('items')[0].get('spec').get(
         'storageDeviceSets')[0].get('count')
     )
+
+
+def get_all_storageclass():
+    """
+    Function for getting all storageclass excluding 'gp2' and 'flex'
+
+    Returns:
+         list: list of storageclass
+
+    """
+    sc_obj = ocp.OCP(
+        kind=constants.STORAGECLASS,
+        namespace=defaults.ROOK_CLUSTER_NAMESPACE
+    )
+    result = sc_obj.get()
+    sample = result['items']
+
+    storageclass = [
+        item for item in sample if (
+            (item.get('metadata').get('name') not in constants.IGNORE_SC_GP2)
+            and (item.get('metadata').get('name') not in constants.IGNORE_SC_FLEX)
```

Comment:
you can change:
    storageclass = [
        item for item in sample if (
            item.get('metadata').get('name') not in (constants.IGNORE_SC_GP2,constants.IGNORE_SC_FLEX)
        )
    ]